### DISCUSSION TITLE
Need help with mouse setting to right stick Xbox series X controller. Problem with right stick Lock center function.

### DISCUSSION THREAD

**Discussion by hongsydney** [x360ce/x360ce#1426] [Q&A]
I tried to use my mouse as look around emulate Right stick of xbox controller. I can set it to work fine lokking around but the problem is it doesn't have Lock center function. When i move my mouse up it stay there and didnt not go back to center of the stick. Does anyone know how to set it properly help me please??
Thank alot.

**Comment by xelmXIV** [comment]
how did you get your mouse to register as the right analog stick? i can't seem to figure it out... all of my inputs are correct and literally everything works EXCEPT my mouse as right analog stick.
i'm basically trying to play an FPS game with Mouse and keyboard [ registered or emulated ] as a controller.

i also didn't see an option for "re-center" but maybe setting a higher DPI on your mouse will allow you to control it easier.
then change your in game sensitivity to something slower. 
I will keep looking around in the program for you.
